npm-crawl.js

```diff
@@ -552,20 +556,17 @@ utils.extend(FetchTask.prototype, {
 		// If a task is currently loading this fileUrl,
 		// wait for it to complete
 		var loadingTask = this.context.loadingPaths[this.fileUrl];
-		if(loadingTask) {
-			var task = this;
-			return loadingTask.promise.then(function(){
-				if(loadingTask.hadErrorLoading()) {
```

Comment:
ok, I just adjusted the conditional, what do you think? is it clear what's going on there?